UI/Pages/Shared/_Layout.cshtml

```diff
@@ -154,9 +161,38 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
             </p>
         </div>
 
+        @if (includeHomeLink)
+        {
+        <div class="govuk-breadcrumbs">
+          <ol class="govuk-breadcrumbs__list">
+            <li class="govuk-breadcrumbs__list-item">
+              <a asp-page="@nameof(Index)" class="govuk-breadcrumbs__link" data-testid="home-link" >Home</a>
+            </li>
+          </ol>
+        </div>
+        }
+        
+        @if (includePrintPage)
```

Comment:
There may be in the future - but not at the moment, all print pages have a back button, so I've done it like this to try and keep the _Layout logic as simple as possible for now.  Should it be needed in the future we'd need to change the _Layout.  Line 14 on _Layout.cshtml makes sure that the print is only shown if there is a back button.

If you'd like we can change it so that they are independent, will be slightly messier in the _Layout.cshtml, but possibly more flexible going forward?

Reply:
I just had a vision of a future us putting a print button on a page and then getting confused as to why it came up with a back link! Maybe just put an `if(includeBackLink)` around the `<div class="app-print-this-page--container-left">` ?